ftw/copymovepatches/testing.py

```diff
@@ -23,11 +25,16 @@ def setUpZope(self, app, configurationContext):
             '</configure>',
             context=configurationContext)
 
-        z2.installProduct(app, 'collective.indexing')
+        if not IS_PLONE_5:
+            z2.installProduct(app, 'collective.indexing')
 
     def setUpPloneSite(self, portal):
         applyProfile(portal, 'ftw.copymovepatches.tests:default')
 
+        if IS_PLONE_5:
+            applyProfile(portal, 'plone.app.contenttypes:default')
+            register_dx_content_builders(force=True)
```

Comment:
Is this necessary?
If I remember correctly, with Plone 5 we should always have DX builders.
Using this function would then only make sense when running Plone 4.

Reply:
Right, we can omit that.